chemicalx/constants.py

```diff
@@ -0,0 +1,11 @@
+"""Constants for ChemicalX."""
+
+from torchdrug.data import Molecule
+from torchdrug.data.feature import atom_default
+
+__all__ = [
+    "TORCHDRUG_NODE_FEATURES",
+]
+
+#: The default number of node features on a molecule in torchdrug
+TORCHDRUG_NODE_FEATURES = len(atom_default(Molecule.dummy_atom))
```

Comment:
Should constants have their own docstring? See this [SO question](https://stackoverflow.com/a/20230473/10478812).